src/main/java/org/folio/circulation/storage/ItemByInstanceIdFinder.java

```diff
@@ -54,7 +53,7 @@ public ItemByInstanceIdFinder(GetManyRecordsClient holdingsStorageClient,
 
       Set<String> holdingsIds = holdingsRecords.toKeys(byId());
 
-      return itemRepository.findByQuery(exactMatchAny("holdingsRecordId", holdingsIds));
+      return itemRepository.findByIndexNameAndHoldingsIds("holdingsRecordId", holdingsIds);
```

Comment:
The name of this method seems strange. The parameters suggest it is intended to be generic, e.g. an index name and some IDs yet the name specifically refers to Holdings IDs.